api/bases/dataplane.openstack.org_openstackdataplaneservices.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.10.0
+  creationTimestamp: null
+  name: openstackdataplaneservices.dataplane.openstack.org
+spec:
+  group: dataplane.openstack.org
+  names:
+    kind: OpenStackDataPlaneService
+    listKind: OpenStackDataPlaneServiceList
+    plural: openstackdataplaneservices
+    singular: openstackdataplaneservice
+  scope: Namespaced
+  versions:
+  - name: v1beta1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            type: string
+          kind:
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              label:
+                type: string
+              role:
+                properties:
+                  any_errors_fatal:
+                    default: true
+                    type: boolean
+                  become:
+                    default: false
+                    type: boolean
+                  gather_facts:
+                    default: false
+                    type: boolean
+                  hosts:
+                    default: '{{ primary_role_name | default([]) }}:{{ deploy_target_host
+                      | default(''overcloud'') }}'
```

Comment:
we shoudl not use the term overcloud in operator or other tripleo terminology.

there is not over and under cloud anymore

Reply:
I'm fine with `default(''overcloud'')` being removed. This seems to be the only open issue for this patch so maybe we can make that change and merge so we can move.